cafehome/models.py

```diff
@@ -1,36 +1,86 @@
 from django.db import models
 from users.models import Customer
 
+processor_choices = [
```

Comment:
create a file choices.py in relevent app and use like this 
constants name are always in upper case 

```
INTEL_I7 = "Intel Core i9"

PROCESSOR_CHOICES = [ 
(INTEL_I7, INTEL_I7), 
....
]
```